src/Microsoft.ML.LightGBM/LightGbmArguments.cs

```diff
@@ -129,7 +129,7 @@ public class Options : ISupportBoosterParameterFactory
                 public double MinChildWeight = 0.1;
 
                 [Argument(ArgumentType.AtMostOnce,
-                    HelpText = "Subsample frequency. 0 means no subsample. "
+                    HelpText = "Subsample frequency for bagging. 0 means no subsample. "
                     + "If subsampleFreq > 0, it will use a subset(ratio=subsample) to train. And the subset will be updated on every Subsample iteratinos.")]
```

Comment:
>subset(ratio=subsample) [](start = 59, length = 23)

What is `subset(ration=subsample)`? Is it a syntax of a language I am not aware of? It's definitely not your problem, but I'd like to raise hands for users.

Reply:
I think its psuedo code to help explain what is happening. I will replace with "it will use a subset to train and the subset will be updated on every subsample iteration" (which is also misspelled.)

---
In reply to: [261474536](https://github.com/dotnet/machinelearning/pull/2808#discussion_r261474536) [](ancestors = 261474536)